src/subscription_manager/managercli.py

```diff
@@ -1478,6 +1479,10 @@ def show_current_release(self):
             print(_("Release not set"))
 
     def _do_command(self):
+        more_product_cert_err_msg = _(
+            "Error: More than one release product certificate installed in /etc/pki/product*. "
+            "There can be installed only one of them."
```

Comment:
Small request: please shorten the message. "Error: More than one release product certificate installed." is enough. 

If someone encounters this, they should ask support (but hopefully only QE will ever see this). Unless the user knows RHSM well, they won't be able to fix the issue by themselves, and if they do know RHSM well, the above suggestion should be sufficient hint.

This is what I meant with the previous round of review feedback. Apologies for not elaborating.